concourse_web_config.tf

```
@@ -7,6 +10,17 @@ locals {
   //  }
   //  )
 
+  web_service_env_vars_saml = var.concourse_saml_conf.enable_saml == true ? {
```

for a bool, you don't need to compare to `true`
```suggestion
  web_service_env_vars_saml = var.concourse_saml_conf.enable_saml ? {
```